cscs-checks/prgenv/helloworld.py

```diff
@@ -16,12 +14,22 @@ class HelloWorldBaseTest(rfm.RegressionTest):
     prgenv_flags = {}
     sourcepath = 'hello_world'
     build_system = 'SingleSource'
+    prebuild_cmds = ['RFM_BUILD_TIME="$(date +%s%N)"']
```

Comment:
I would change the environment variable name, because this could conflict with a future reframe environment variable: `_rfm_build_time` is better, but isn't there a way to just prepend `time` to the compilation command by hacking the `build_job`? I think that you could so something like this:

```python
@run_before('compile')
def time_compile(self):
    self.build_job.launcher = LauncherWrapper(self.build_job.launcher, 'time')
```

Reply:
I have only updated the variable so far. I think it's a better solution than `time` just in case the compilation takes more than one command to build. Not sure if this is the case for any buildsystem and I think in this test it's not relevant but I think in general it's safer to do it like this. If you still think it's better to change the launcher command I will update.